themes/pom.xml

```diff
@@ -16,6 +16,11 @@
         <dir.common>src/main/resources/theme/keycloak/common/resources</dir.common>
         <dir.account2>src/main/resources/theme/keycloak.v2/account/src</dir.account2>
         <args.npm.install>ci --no-optional --ignore-scripts</args.npm.install>
+        <!-- ignore folders for incremental builds by the maven build cache plugin -->
+        <maven.build.cache.exclude.1>${project.basedir}/src/main/resources/theme/keycloak/common/resources/node_modules</maven.build.cache.exclude.1>
+        <maven.build.cache.exclude.2>${project.basedir}/src/main/resources/theme/keycloak/common/resources/web_modules</maven.build.cache.exclude.2>
+        <maven.build.cache.exclude.3>${project.basedir}/src/main/resources/theme/keycloak.v2/account/src/node_modules</maven.build.cache.exclude.3>
+        <maven.build.cache.exclude.4>${project.basedir}/src/main/resources/theme/keycloak.v2/account/src/web_modules</maven.build.cache.exclude.4>
```

Comment:
Why are these being excluded? Is it to speed up the hashing? I'd say in general it would be useful if updates to the deps caused a rebuild.

Reply:
The node_modules are filled by npm with the downloaded and expanded packages, and the web_modules are compile/expanded contents as well. They appear during build time, and they are ignored using `.gitignore`. 

Therefore I need to exclude them, as they should not be part of the hash, as otherwise the artifact would not be reused.

Reply:
Guess this is okay as I'd imagine these wouldn't be updated unless for example there is an update to a `package.json` or something, which would trigger a re-build. We can always adjust later if needed.